src/SmartComponents/BaselinesTable/BaselinesToolbar/__test__/__snapshots__/BaselinesToolbar.test.js.snap

```diff
@@ -202,14 +202,14 @@ exports[`jest-tests ConnectedBaselinesToolbar should render correctly 1`] = `
             Array [
               "1234",
               "beavs baseline",
-              "3 years ago",
+              "4 years ago",
```

Comment:
I'm not sure if this should be in this PR. It doesn't affect the overlapping, or?

Reply:
@bastilian It's a testing thing. We have a column in baselines that displays when a baseline was last updated. So, unless someone wants to provide me with a way to dynamically code the fixtures, we'll have to update these tests every year.

Reply:
Either way, it should be included for this PR, but we shouldn't add a fix in this PR, if that makes sense.